File: C/평균은 넘겠지.c
#include <stdio.h>

int main() {
    int n, m, i, j, sum, score[1000];
    float avg;
    
    scanf("%d", &n);
    for (i = 0; i < n; i++){
        scanf("%d", &m);
        sum = 0;
        for (j = 0; j < m; j++){
            scanf("%d", &score[j]);
            sum += score[j];
        }
        avg = (float)sum/(float)m;
        sum = 0;
        for (j = 0; j < m; j++){
            if(avg < score[j]){
                sum++;
            }
        }
        avg = (float)sum/(float)m*100;
        printf("%.3f%%\n", avg);
    }
    return 0;
}

File: Python/평균은 넘겠지.py
n = int(input())
for _ in range(n):
    score = list(map(int, input().split()))
    avg = sum(score[1:]) / score[0]
    sm = 0
    for i in score[1:]:
        if i > avg:
            sm += 1
    avg = sm/score[0]*100
    
    print('{:.3f}%'.format(avg))
